src/entities.py

```diff
@@ -118,18 +118,29 @@ def to_json_obj(self):
 
 class Canteen(ApiRepresentable, Enum):
     # Some of the canteens do not use the general Studentenwerk system and therefore do not have a url_id.
-    def __init__(self, long_name: str, location: Location, url_id: int):
+    def __init__(self, long_name: str, location: Location, url_id: int, queue_status: str):
```

Comment:
This should be fine for now. If everything goes like planed, we will soon have a general API for all canteens utilizing the LRZ/RBG access point statistics for that.

Reply:
Okay, that sounds cool.
Is there already something planed, where one can see progress or even contribute to that?